paasta_tools/cli/cmds/check.py

```diff
@@ -288,15 +299,15 @@ def sensu_check(service, service_path):
         print PaastaCheckMessages.SENSU_MONITORING_MISSING
 
 
-def service_dir_check(service):
+def service_dir_check(service_name, service_root=DEFAULT_SOA_DIR):
```

Comment:
Now that we have the big default in the args, can you remove the kwarg default here to prevent someone from making the mistake of *not* passing service_root, causing a regression?